### DISCUSSION TITLE
Why is the NSAllowsArbitraryLoads required?

### DISCUSSION THREAD

**Discussion by dryprogrammer** [fluttercandies/flutter_wechat_assets_picker#499] [General]
Hello, new user here. I am just wondering why is the NSAllowsArbitraryLoads required for this package? This is for unsecured HTTP and may make App Store review more difficult.
https://github.com/fluttercandies/flutter_wechat_assets_picker#ios says NSAllowsArbitraryLoads is required.
https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

**Comment by AlexV525** [comment]
Looks like a mistake when writing docs. Thanks for the report.
